api/tests/unit/domain/services/solution-service-qrocm-ind_test.js

```diff
@@ -4,320 +4,1083 @@ const service = require('../../../../lib/domain/services/solution-service-qrocm-
 
     const allCases = [
-      {when:'no stress',                   output: 'ok', answer: '9lettres: courgette\n6lettres: chicon',        solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- chicon\n- legume\n',      deactivations: {t1:true}},
-      {when:'spaces stress',               output: 'ko', answer: '9lettres: courgette\n6lettres: c h i c o n',   solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- chicon\n- legume\n',      deactivations: {t1:true}},
-      {when:'reverted spaces stress',      output: 'ko', answer: '9lettres: courgette\n6lettres: chicon',        solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n-  c h i c o n \n- legume\n', deactivations: {t1:true}},
-      {when:'uppercase stress',            output: 'ko', answer: '9lettres: courgette\n6lettres: CHICON',        solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- chicon\n- legume\n',      deactivations: {t1:true}},
-      {when:'reverted uppercase stress',   output: 'ko', answer: '9lettres: courgette\n6lettres: chicon',        solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- CHICON\n- legume\n',      deactivations: {t1:true}},
-      {when:'accent stress',               output: 'ko', answer: '9lettres: courgette\n6lettres: îàéùô',         solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- iaeuo\n- legume\n',       deactivations: {t1:true}},
-      {when:'reverted accent stress',      output: 'ko', answer: '9lettres: courgette\n6lettres: iaeuo',         solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- îàéùô\n- legume\n',       deactivations: {t1:true}},
-      {when:'diacritic stress',            output: 'ko', answer: '9lettres: courgette\n6lettres: ççççç',         solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- ccccc\n- legume\n',       deactivations: {t1:true}},
-      {when:'reverted diacritic stress',   output: 'ko', answer: '9lettres: courgette\n6lettres: ccccc',         solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- ççççç\n- legume\n',       deactivations: {t1:true}},
-      {when:'punctuation stress',          output: 'ok', answer: '9lettres: courgette\n6lettres: .!p-u-n-c-t',   solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- punct\n- legume\n',       deactivations: {t1:true}},
-      {when:'reverted punctuation stress', output: 'ok', answer: '9lettres: courgette\n6lettres: punct',         solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- .!p-u-n-c-t\n- legume\n', deactivations: {t1:true}},
-      {when:'levenshtein stress',          output: 'ok', answer: '9lettres: courgette\n6lettres: 0123456789',    solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- 123456789\n- legume\n',   deactivations: {t1:true}},
-      {when:'reverted levenshtein stress', output: 'ok', answer: '9lettres: courgette\n6lettres: 123456789',     solution: '9lettres:\n- courgette\n6lettres:\n- tomate\n- 0123456789\n- legume\n',  deactivations: {t1:true}},
+      {
```

Comment:
de même 'reverted' dans les descriptions n'était pas très explicite. Il signifiait que les traitements se faisaient sur les solutions possibles => donc 'reverted' => 'on the solution'